### ISSUE TITLE
Errors and missing sensors after wake-up

### ISSUE THREAD

**Issue by johny-mnemonic** [Rem0o/FanControl.Releases#489]
I am using FanControl for a really long time and it always worked wonders, but for last two months I started getting errors after waking up my PC.
Error log contains the following:
```
25.01.2022 22:52:25: System.Management.ManagementException: Timed out 
   v System.Management.ManagementException.ThrowWithExtendedInfo(ManagementStatus errorCode)
   v System.Management.ManagementObjectCollection.ManagementObjectEnumerator.MoveNext()
   v LibreHardwareMonitor.Hardware.Storage.StorageGroup..ctor(ISettings settings)
   v LibreHardwareMonitor.Hardware.Computer.AddGroups()
   v LibreHardwareMonitor.Hardware.Computer.Open()
   v FanControl.Domain.BackendProviders.LHM.LHMBackendProvider.Open()
25.01.2022 22:52:25: LibreHardwareMonitor could not initialize.
25.01.2022 22:52:25: Unhandled exception in FanControl v1.0.0.0
25.01.2022 22:52:25: System.AggregateException: Výjimky úlohy nebyly pozorovány při čekání na úlohu ani při přístupu k vlastnosti Exception. Následkem toho finalizační vlákno znovu vygenerovalo nepozorovanou výjimku. ---> System.InvalidOperationException: Volající vlákno nemůže získat přístup k tomuto objektu, protože jej vlastní jiné vlákno.
   v System.Windows.Threading.Dispatcher.VerifyAccess()
   v FanControl.Domain.ApplicationContext.ShowMainWindow()
   v FanControl.ViewModels.MainWindowViewModel.<ShowError>d__144.MoveNext()
   --- Konec trasování zásobníku pro vnitřní výjimku ---
---> (Inner Exception #0) System.InvalidOperationException: Volající vlákno nemůže získat přístup k tomuto objektu, protože jej vlastní jiné vlákno.
   v System.Windows.Threading.Dispatcher.VerifyAccess()
   v FanControl.Domain.ApplicationContext.ShowMainWindow()
   v FanControl.ViewModels.MainWindowViewModel.<ShowError>d__144.MoveNext()<---

25.01.2022 22:52:26: System.InvalidOperationException: Volající vlákno nemůže získat přístup k tomuto objektu, protože jej vlastní jiné vlákno.
   v System.Windows.Threading.Dispatcher.VerifyAccess()
   v FanControl.Domain.ApplicationContext.ShowMainWindow()
   v FanControl.ViewModels.MainWindowViewModel.<ShowError>d__144.MoveNext()
```
I have Windows in Czech and that stupid thing translates error messages to Czech language🙄. Translation of the most common error "Volající vlákno nemůže získat přístup k tomuto objektu, protože jej vlastní jiné vlákno." is something like "Calling thread can't get access to the object as it is owned by different thread"

UI produces this popup:
![fancontrol_wakeup_error](https://user-images.githubusercontent.com/1143074/151067337-861e2060-ee7f-4906-b620-411e1e43f4b9.png)
Sometimes the "Refresh" button works, sometimes it doesn't and in that case I have to kill FanControl using task manager as it gets completely stuck in a way where UI seems to work, but anything you click doesn't actually do anything.

I am not aware of changing HW config of my machine, so I guess either something changed in FanControl/LHM or in Windows 10 due to some system update...
My system specs:
MSI B450M Mortar MAX
Ryzen R5 1600AF
32GB DDR4 3200
Radeon RX 6800
A-Data SX8200 PRO 1TB (2x)

**Comment by redstonewhite** [comment]
I wonder whether it helps. I have the same problem with you and everytime I reboot my computer it will tell me "missing sensor"

**Comment by johny-mnemonic** [comment]
@redstonewhite It helped, but you seem to have different issue than me.
I am now pretty certain it was caused by an external USB HDD I have connected to my PC to create a backup. I haven't realized this as an HW change, but it perfectly matches with the start of my issues. Apparently that USB controller is slow to wake-up after sleep and causes timeout for LHM.
@Rem0o Thanks for the hint again!
